### ISSUE TITLE
Publish latest release to Hackage?

### ISSUE THREAD

**Issue by ozkutuk** [jonschoning/espial#33]
Hi! I am an Espial user on NixOS, and I am trying to write a NixOS module for Espial to make it easier for people on NixOS to run an Espial service. However, [nix package repository](https://github.com/NixOS/nixpkgs/) grabs Haskell packages off of Hackage, and the currently published version of Espial (`0.0.7`) fails to build with current Nix-Haskell infrastructure. Therefore the package is marked as broken. I have tested building it with the latest source on GitHub and it seems to build fine.

TL;DR: Is it possible to publish the latest release version (which seems to be `0.0.9`) to Hackage? Thanks!

**Comment by jonschoning** [comment]
created https://hackage.haskell.org/package/espial-0.0.9
